From 31693d02b06ed2f0ac668cede16cf3258e86204e Mon Sep 17 00:00:00 2001
From: Colin Ian King <colin.i.king@gmail.com>
Date: Thu, 21 Apr 2022 09:55:46 +0100
Subject: [PATCH] net: hns3: Fix spelling mistake "actvie" -> "active"
Git-commit: 31693d02b06ed2f0ac668cede16cf3258e86204e
Patch-mainline: v5.19-rc1
References: bsn#12
Modified-by-SEL: No


There is a spelling mistake in a netdev_info message. Fix it.

Signed-off-by: Colin Ian King <colin.i.king@gmail.com>
Link: https://lore.kernel.org/r/20220421085546.321792-1-colin.i.king@gmail.com
Signed-off-by: Jakub Kicinski <kuba@kernel.org>
Signed-off-by: Guoqing Jiang <guoqing.jiang@suse.com>
---
 drivers/net/ethernet/hisilicon/hns3/hns3_ethtool.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/net/ethernet/hisilicon/hns3/hns3_ethtool.c b/drivers/net/ethernet/hisilicon/hns3/hns3_ethtool.c
index bb001f597857..1db8a86f046d 100644
--- a/drivers/net/ethernet/hisilicon/hns3/hns3_ethtool.c
+++ b/drivers/net/ethernet/hisilicon/hns3/hns3_ethtool.c
@@ -1915,7 +1915,7 @@ static int hns3_set_tunable(struct net_device *netdev,
 			return ret;
 		}
 
-		netdev_info(netdev, "the actvie tx spare buf size is %u, due to page order\n",
+		netdev_info(netdev, "the active tx spare buf size is %u, due to page order\n",
 			    priv->ring->tx_spare->len);
 
 		break;
-- 
2.34.1

